examples/sveltekit-pwa/pwa-configuration.js

```diff
@@ -85,11 +63,9 @@ if (sw) {
 	pwaConfiguration.strategies = 'injectManifest'
 	pwaConfiguration.manifest.name = 'PWA Inject Manifest'
 	pwaConfiguration.manifest.short_name = 'PWA Inject'
-	pwaConfiguration.injectManifest = workboxOrInjectManifestEntry
+	pwaConfiguration.svelteKitVitePluginOptions = svelteKitVitePluginOptions
```

Comment:
since this is being done in both the `if` and `else` it could be moved outside the `if`/`else`